test/hotspot/jtreg/compiler/c2/irTests/XorINodeIdealizationTests.java

```diff
@@ -57,7 +59,7 @@ public void runMethod() {
         int d = RunInfo.getRandom().nextInt();
 
         int min = Integer.MIN_VALUE;
-        int max = Integer.MAX_VALUE;
+        int max = MAX_VALUE;
```

Comment:
Let's make it consistent, and either use `Integer.` everywhere or without it everywhere ;)